engine/openbor.h

```diff
@@ -1790,8 +1790,6 @@ if(n<1) n = 1;
 //#define     MAX_MOVES             16
 //#define     MAX_MOVE_STEPS        16
 
-#pragma pack(4)
```

Comment:
Have to be-careful with packing.  If data structures are not written/read from disk or being transferred over communication protocols this flag can cause bad behaviors.  I encountered an issue where the pointer being assigned with an array of 11 elements was fragment and yet the code is expecting contiguous memory storage.... mostly arm based devices would have trouble with this.